src/main/java/com/github/tomakehurst/wiremock/client/HttpAdminClient.java

```diff
@@ -132,6 +137,25 @@ public HttpAdminClient(
     httpClient = HttpClientFactory.createClient(createProxySettings(proxyHost, proxyPort));
   }
 
+  public HttpAdminClient(
+      String scheme,
+      String host,
```

Comment:
can the constructor above delegate down to this new one.

```java
    this(
        scheme,
        host,
        port,
        urlPathPrefix,
        hostHeader,
        authenticator,
        HttpClientFactory.createClient(createProxySettings(proxyHost, proxyPort)));
```
should do the trick.